.github/workflows/ssm-command.yaml

```diff
@@ -0,0 +1,452 @@
+                for (const instance of reservation.Instances || []) {
+                  if (instance.InstanceId && instance.PublicDnsName && instance.KeyName) {
+                    instances.push({
+                      instanceId: instance.InstanceId,
+                      publicDns: instance.PublicDnsName,
+                      keyName: instance.KeyName,
+                      name: instance.Tags?.find(tag => tag.Key === 'Name')?.Value || 'unknown'
+                    });
+                  }
+                }
+              }
+            } else {
+              console.log(`🔍 SSH: Discovering all running agent instances...`);
+              const describeParams = {
+                Filters: [
+                  {
+                    Name: 'tag:Name',
+                    Values: ['vargas-jr*', 'pr-*']
+                  },
+                  {
+                    Name: 'instance-state-name',
+                    Values: ['running']
+                  }
+                ]
+              };
+              
+              const result = await ec2.send(new DescribeInstancesCommand(describeParams));
+              
+              for (const reservation of result.Reservations || []) {
+                for (const instance of reservation.Instances || []) {
+                  if (instance.InstanceId && instance.PublicDnsName && instance.KeyName) {
+                    const nameTag = instance.Tags?.find(tag => tag.Key === 'Name')?.Value || 'unknown';
+                    console.log(`📍 SSH: Found instance: ${instance.InstanceId} (${nameTag})`);
+                    instances.push({
+                      instanceId: instance.InstanceId,
+                      publicDns: instance.PublicDnsName,
+                      keyName: instance.KeyName,
+                      name: nameTag
+                    });
+                  }
+                }
+              }
+            }
+            
+            if (instances.length === 0) {
+              console.log(`❌ SSH: No target instances found`);
+              if (targetInstanceName) {
+                console.log(`   Searched for instance named: ${targetInstanceName}`);
+              } else {
+                console.log(`   Searched for running agent instances (vargas-jr*, pr-*)`);
+              }
+              return JSON.stringify({ success: false, message: "No instances found" });
+            }
+            
+            console.log(`🚀 SSH: Executing command on ${instances.length} instance(s): ${command}`);
+            
+            const results = [];
+            
+            // Execute command on all found instances via SSH
+            for (const instance of instances) {
+              console.log(`\n🔗 SSH: Connecting to ${instance.name} (${instance.instanceId})...`);
+              
+              try {
+                // Create temporary key file
+                const keyPath = `/tmp/${instance.keyName}.pem`;
+                const keyContent = process.env.SSH_PRIVATE_KEY;
```

Comment:
We should use kms to fetch our key content. We should already have this logic in the scripts directory